homepage.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+"net/http"
+)
+
+func startUiMode() (int) {
+	mux := http.NewServeMux()
+	mux.Handle("/", http.FileServer(http.Dir("./ui")))
```

Comment:
> As a special case, the returned file server redirects any request ending in "/index.html" to the same path, without the final "index.html".

We don't want the user to have to type index.html. So '/' should serve the index.html file without specifying a file name. Consider that users might make typos etc. and we could have other files in that directory we don't want to expose.